nexusutils/dataconverter/readers/em_spctrscpy/utils/em_example_data.py

```diff
@@ -0,0 +1,416 @@
+        """Copy data in user section."""
+        # check if required fields exists and are valid
+        # print("Parsing user...")
+        prefix = "/ENTRY[entry" + str(self.entry_id) + "]/"
+        user_names = np.unique(
+            np.random.choice(["Sherjeel", "MarkusK", "Benedikt", "Johannes",
+                              "Gerardo", "Kristiane", "Sabine", "Sophie", "Tom",
+                              "Volker", "MarkusW", "PeterK", "Oonagh", "Annika",
+                              "ChristophP", "Thomas", "Mariano", "Tilmann",
+                              "ChristophF", "Niels", "Dieter", "Alexander",
+                              "Katharina", "Florian", "Sebastian", "Sandor",
+                              "Carola", "Chris", "Hampus", "Pepe", "Lauri",
+                              "MarkusS", "Christoph", "Claudia"],
+                             1 + np.random.choice(MAX_USERS, 1)))
+        user_id = 1
+        for name in user_names:
+            trg = prefix + "USER[user" + str(user_id) + "]/"
+            template[trg + "name"] = str(name)
+            user_id += 1
+        return template
+
+    def emulate_sample(self, template: dict) -> dict:
+        """Copy data in specimen section."""
+        # check if required fields exists and are valid
+        # print("Parsing sample...")
+        trg = "/ENTRY[entry" + str(self.entry_id) + "]/sample/"
+        template[trg + "method"] = "simulation"
+
+        self.elements_observed \
+            = np.random.choice(np.asarray(np.linspace(1, 94, num=118, endpoint=True),
+                                          np.uint32),
+                               1 + int(np.random.uniform(low=0, high=5)))
+        assert len(self.elements_observed) > 0, "List of assumed elements is empty!"
+        unique_elements = set()
+        for atomic_number in self.elements_observed:
+            symbol = chemical_symbols[atomic_number]
+            assert isinstance(symbol, str), "symbol is not a string!"
+            if (symbol in chemical_symbols) & (symbol != "X"):
+                unique_elements.add(symbol)
+        print("Unique elements are")
+        print(list(unique_elements))
+        template[trg + "atom_types"] = list(unique_elements)
+
+        specimen_name = "Mocked electron microscopy specimen " \
+            + str(np.random.choice(1000, 1)[0])
+        template[trg + "name"] = specimen_name
+        template[trg + "sample_history"] = "n/a"
+        template[trg + "preparation_date"] \
+            = datetime.datetime.now().astimezone().isoformat()
+        template[trg + "short_title"] \
+            = specimen_name.replace("Mocked atom probe specimen ", "")
+        template[trg + "description"] = "n/a"
+
+        template[trg + "thickness"] \
+            = np.float64(np.max((np.random.normal(loc=40., scale=5.0), 10.)))
+        template[trg + "thickness/@units"] = "nm"
+        template[trg + "density"] = 0.  # is optional
```

Comment:
type here. should be "= 0.0" It translates to a float when you do "0." as well. But seems quite odd.